src/Raven.Studio/typescript/common/copyToClipboard.ts

```diff
@@ -1,6 +1,16 @@
 ﻿import messagePublisher = require("common/messagePublisher");
 
 class copyToClipboard {
+
+    constructor(private htmlElement: HTMLElement, private textArray: Array<string>) {
```

Comment:
why do we need html element here and some list?

Reply:
done - refactored.